javaLoginCore/src/main/java/com/hamdan/forzenbook/java/login/core/data/repository/LoginRepositoryImpl.java

```diff
@@ -39,8 +40,8 @@ private User getTokenFromDatabase() {
     }
 
     @NonNull
-    private User getTokenFromNetwork(String email, String code) throws FailTokenRetrievalException, NullTokenException {
-        Response<LoginResponse> response = service.getToken(email, code);
+    private User getTokenFromNetwork(String email, String code) throws FailTokenRetrievalException, NullTokenException, IOException {
+        Response<LoginResponse> response = service.getToken(email, code).execute();
```

Comment:
You wrapped this in call just to call .execute lol you should only need Call in the weird no body case, no? Or is Java being weird?

Reply:
Java being weird as fuck. Useing response just made it refuse to even want to work.